cms/templates/component.html

```diff
@@ -21,8 +21,11 @@
       </li>
       <li class="action-item action-move">
           <button data-tooltip="${_("Move")}" class="btn-default move-button action-button">
-            <span class="icon fa fa-folder-o" aria-hidden="true"></span>
-            <span class="sr">${_("Move")}</span>
+              <span class="stack-move-icon fa-stack fa-lg">
```

Comment:
I think we should have some sort of condition when to show move and when to not show ? like in case of library instance this is visible move modal button.

@muzaffaryousaf thoughts?